src/Compilers/Core/RebuildTest/RebuildCommandLineTests.cs

```diff
@@ -112,6 +147,8 @@ void Method()
 
             Add(Permutate(new CommandInfo("hw.cs /target:exe /debug:embedded", "test.exe", null)));
             Add(Permutate(new CommandInfo("lib1.cs /target:library /debug:embedded", "test.dll", null)));
+            Add(Permutate(new CommandInfo("lib2.cs /target:library /r:SystemRuntime=System.Runtime.dll /debug:embedded", "test.dll", null)));
+            Add(Permutate(new CommandInfo("lib3.cs /target:library /r:SystemRuntime1=System.Runtime.dll /r:SystemRuntime2=System.Runtime.dll /debug:embedded", "test.dll", null)));
```

Comment:
>/r:SystemRuntime2=System.Runtime.dll [](start = 104, length = 36)

Would it make sense to reference two different versions of System.Runtime.dll, so it's clear that references in the rebuilt binary are to the expected versions?

Reply:
The case I'm interested in testing here is when it refers to the same DLL